NEWS.md

```diff
@@ -1,80 +1,116 @@
 # ggplot2 3.1.0.9000
 
-* `geom_rug()` gains a `length` option to allow for changing the length of the rug lines. (@daniel-wells, #3109)
+This is a minor release with a main emphasis on internal changes to make ggplot2 
+faster and more consistent. The few breaking changes there are will only affect
+the aesthetics of the plot (but not break code) or, potentially, extension 
+developers if they have relied on internals that have been changed.
 
-* `coord_sf()` graticule lines are now drawn in the same thickness as 
-  panel grid lines in `coord_cartesian()`, and seting panel grid
-  lines to `element_blank()` now also works in `coord_sf()` 
+## Breaking Changes
+
+* Default labels are now generated more consistently; e.g., symbols no longer
```

Comment:
Why is this a breaking change?

Reply:
It changes the visual of some plots... I agree it is a slim one